src/App.scss

```diff
@@ -0,0 +1,203 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@700&display=swap");
+
+:root {
+  //fonts
+  --Monserrat: "Montserrat", sans-serif;
+  --Roboto: "Roboto Slab", serif;
+}
+
+body {
+  font-family: var(--Monserrat);
+  background-color: #fafafa;
+}
+
+.toptitle {
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
+  font-size: 1.875rem;
+}
+
+.head-links {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.toplinks {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  gap: 15px;
+}
+
+.toplinks li a {
+  font-family: var(--Monserrat);
+  width: 3.688rem;
+  height: 1rem;
+  color: #12121283;
+  text-decoration: none;
+  font-size: 0.813rem;
+}
+
+.toplinks li a:hover {
+  color: #121212;
+  font-weight: 800;
+}
+
+.separator {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.user {
+  height: 25px;
+  width: 20px;
+}
+
+// Book
+
+.main-book-container {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  border: 1px solid #8080803d;
+  padding: 15px;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 15px;
+}
+
+.btncont {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.everyBook {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+}
+
+.spcategory {
+  font-size: 0.875rem;
+  font-weight: bold;
+  opacity: 0.5;
+}
+
+.sptitle {
+  font-family: var(--Roboto);
+  font-size: 1.375rem;
+  font-weight: bold;
+}
+
+.spauthor {
+  font-family: var(--Roboto);
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
```

Comment:
- [ ] Since you have started using CSS variables, it would be nice to use it for CSS rules (color: #4386bf;) like this that are repeated. Kindly apply this change to other CSS rules that are repeated in this file. 👍

Reply:
Thanks for the suggestion, added the colors that were used more than once to the variable list!